src/Parsers/ASTAlterQuery.cpp

```diff
@@ -11,6 +11,14 @@ namespace ErrorCodes
     extern const int UNEXPECTED_AST_STRUCTURE;
 }
 
+String ASTAlterCommand::getID(char delim) const
+{
+    String id = "AlterCommand";
+    id.append(1, delim);
```

Comment:
Consider to use operator + here, it will have special overload for `char`.

Reply:
Done.